backends/advanced/src/advanced_omi_backend/memory/memory_service.py

```
@@ -461,608 +109,659 @@ async def add_memory(
+                memory_logger.info(f"💾 No memories extracted, storing raw transcript for {audio_uuid}")
+
+            memory_logger.debug(f"🧠 fact_memories_text: {fact_memories_text}")
+            # Simple deduplication of extracted memories within the same call
+            fact_memories_text = self._deduplicate_memories(fact_memories_text)
+            memory_logger.debug(f"🧠 fact_memories_text after deduplication: {fact_memories_text}")
+            # Generate embeddings
+            embeddings = await asyncio.wait_for(
+                self.llm_provider.generate_embeddings(fact_memories_text),
+                timeout=self.config.timeout_seconds
+            )
+            memory_logger.info(f"embeddings generated")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)